Smartsheet.Net.Standard/Entities/CellHistory.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Smartsheet.Net.Standard.Interfaces;
+
+namespace Smartsheet.Net.Standard.Entities
+{
+    public class CellHistory : ISmartsheetObject
+    {
+        public long? ColumnId { get; set; }
+        public string DisplayValue { get; set; }
+        public dynamic Value { get; set; }
+        public DateTime ModifiedAt { get; set; }
+        public ContactOptions ModifiedBy { get; set; }
```

Comment:
The API docs claim this is a full `User` object but actual results show it being only a name and email. I'm verifying with Kevin Fransler. What you have here is probably correct, but I'm going to wait for a response to make sure this shouldn't actually be a full User object.

Reply:
Left as is for now until Kevin says otherwise.

Reply:
Kevin is sticking with it being a "lightweight" version of the User object, rather than a different object altogether, and the public C# SDK implements it as a full User object. Despite that I personally believe it's more clear to make the distinction by treating it as a different object altogether, I do think it's best to match the documentation, so I'm going to say this should be `User` rather than `ContactOptions`. I think the _right_ fix would be to change the documentation but the other SDKs already treat this is a `User` object so that would come with more overhead than it's probably worth.

Reply:
Updated to match the User object. While I understand it's a "lightweight" version, it's confusing from a developer perspective when it returns only the name and email of that user, which happens to be what ContactOptions is. 

![image](https://user-images.githubusercontent.com/45579436/63362346-49c86c00-c337-11e9-9af4-2b49dc902bf3.png)